Containerfile.cuda

```diff
@@ -0,0 +1,19 @@
+FROM fedora:39
+# make sure nvidia driver match on host and in the container
+RUN dnf install -y python3-requests python3-pip gcc gcc-c++ python3-scikit-build git-core \
+    && echo "[cuda-fedora39-x86_64]" >> /etc/yum.repos.d/cuda.repo \
+    && echo "name=cuda-fedora39-x86_64" >> /etc/yum.repos.d/cuda.repo \
+    && echo "baseurl=https://developer.download.nvidia.com/compute/cuda/repos/fedora39/x86_64" >> /etc/yum.repos.d/cuda.repo \
+    && echo "enabled=1" >> /etc/yum.repos.d/cuda.repo \
+    && echo "gpgcheck=1" >> /etc/yum.repos.d/cuda.repo \
+    && echo "gpgkey=https://developer.download.nvidia.com/compute/cuda/repos/fedora39/x86_64/D42D0685.pub" >> /etc/yum.repos.d/cuda.repo \
```

Comment:
Wouldn't it be better to just have .repo file next to the Containerfile and COPY it into the container. It's seems simpler.

Reply:
yes it would, but your [comment](https://github.com/fedora-copr/logdetective/pull/51#issuecomment-2277488054) would become even more relevant :) I agree I don't want to flood the root dir of this repo with "random" files

Reply:
I really like that now we can see the content of the repo file and i.e. once F40 variant is available, we can `s/39/40/g` in one file